disco_aws_automation/disco_alarm_config.py

```diff
@@ -50,6 +49,11 @@ def __init__(self, options, maximum=True):
         self.custom_metric = options["custom_metric"].lower() == "true"
         self.log_pattern_metric = disco_aws_util.is_truthy(options.get("log_pattern_metric", ""))
 
+        if "autoscaling_group_name" in options:
```

Comment:
Could also do
`self.autoscaling_group_name = options["autoscaling_group_name"] if "autoscaling_group_name" in options else None`
though maybe that's not easier to parse 😝.